articles/azure/azs-how-create-scale-set.md

```diff
@@ -0,0 +1,126 @@
+---
+title: How to Create a Scale Set using UKCloud Azure Stack portal | UKCloud Ltd
+description: Provides help for creating a scale set on UKCloud for Microsoft Azure
+services: azure-stack
+author: Shaion O'Niel
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a Scale Set - portal
```

Comment:
Portal with a capital letter in TOC
Scale Set should be lowercase scale set literally everywhere.